docs/src/01adr-cli.md

```diff
@@ -0,0 +1,107 @@
+# ADR-01: Atomkraft CLI
+
+| authors           | revision | revision date  |
+| ----------------- | --------:| --------------:|
+| Andrey Kuprianov  |        1 | July 11, 2022  |
+
+This ADR describes the command-line interface of Atomkraft, and how it is supposed to be used by the users as a standalone tool. It *does not* describe other possible usage scenarios, e.g. integration with testing frameworks (such as Pytest). We structure the description around the tool use cases.
+
+Below we specify only the outcomes for successful command execution. Upon unsuccessful command execution, the error should be reported to the user, and no remnants (e.g. zombie processes, or additional files beyond requested) should remain.
+
+## General principles
+
+The tool CLI commands represent top-level user workflows, where each workflow is able to bring value to a user. Our primary user for the CLI is the auditor, who comes to a new Cosmos-SDK based project, and wants to test it in a limited amount of time.
+
+Each CLI command produces a certain artifact (e.g. a blockchain configuration, or a Python reactor stub). Those intermediate artifacts can be customized, and then seamlessly integrated back in the workflows that depend on them.
+
+An example workflow with customization points:
+
+1. A user comes to the new blockchain. They want to check that Atomkraft is able to interact with it, so they run `atomkraft testnet ...`
```

Comment:
Do we really need to launch a testnet separately? Testnet should be auto launched when `atomkraft test` is run and shutdown (or stored in some log directory) when it is finished.

Reply:
If this is about using `explorer` on running testnet, we can store the testnet and respawn the testnet when someone wants to inspect.